SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialWorkerType.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
```

Comment:
coreminimal seems pretty here for such a small file, can we explicitly declare the reqs?